releases/efs-provisioner.yaml

```
@@ -0,0 +1,66 @@
+repositories:
+# Stable repo of official helm charts
+- name: "stable"
+  url: "https://kubernetes-charts.storage.googleapis.com"
+
+releases:
+
+################################################################################
+## EFS provisioner #############################################################
+################################################################################
+
+#
+# References:
+#   - https://github.com/helm/charts/blob/master/stable/efs-provisioner/values.yaml
+#   - https://github.com/kubernetes-incubator/external-storage/tree/master/aws/efs
+#
+- name: "efs-provisioner"
+  namespace: "codefresh"
+  labels:
+    chart: "efs-provisioner"
+    repo: "stable"
+    component: "codefresh"
+    namespace: "codefresh"
+    vendor: "kubernetes-incubator"
+    default: "true"
```

```suggestion
    default: "false"
```

We prob don't want to always provision it by default